### PR TITLE
[CL][Internal Review] Write test for getUptimeTrackerValues

### PR DESCRIPTION
**Pull request by stackman27**
Closes: [#5088](https://github.com/osmosis-labs/osmosis/issues/5088)

## What is the purpose of the change

test is missing for getUptimeTrackerValues in incentives.go

## Brief Changelog
n/a

## Testing and Verifying
added test 

## Documentation and Release Note

  - Does this pull request introduce a new feature or user-facing behavior changes? (yes / no)
  - Is a relevant changelog entry added to the `Unreleased` section in `CHANGELOG.md`? (yes / no)
  - How is the feature or change documented? (not applicable   /   specification (`x/<module>/spec/`)  /  [Osmosis docs repo](https://github.com/osmosis-labs/docs)   /   not documented)

### CODE REVIEW

File: x/concentrated-liquidity/export_test.go [hunk 1]
```diff
@@ -227,7 +227,7 @@ func (k Keeper) CollectIncentives(ctx sdk.Context, owner sdk.AccAddress, positio
 	return k.collectIncentives(ctx, owner, positionId)
 }
 
-func GetUptimeTrackerValues(uptimeTrackers []model.UptimeTracker) []sdk.DecCoins {
+func (k Keeper) GetUptimeTrackerValues(uptimeTrackers []model.UptimeTracker) []sdk.DecCoins {
```
Comment (p0mvn) [inline x/concentrated-liquidity/export_test.go:230]: What is the reason for adding this? Can this be reverted to avoid confusion?

**Review by p0mvn — APPROVED** [review]

File: x/concentrated-liquidity/export_test.go (hunk 1 above)
Comment (czarcas7ic) [inline x/concentrated-liquidity/export_test.go:230]: Why was this changed as a keeper when the non exported method is not a keeper method?

Comment (stackman27) [reply x/concentrated-liquidity/export_test.go]: oh yea sorry for the confusion

Comment (stackman27) [reply x/concentrated-liquidity/export_test.go]: oh yea sorry for the confusion
